webservices/common/models.py

```diff
@@ -556,3 +556,79 @@ class CommitteeTotalsHouseSenate(CommitteeTotals):
     other_receipts = db.Column(db.Integer)
     transfers_from_other_authorized_committee = db.Column(db.Integer)
     transfers_to_other_authorized_committee = db.Column(db.Integer)
+
+
+class ScheduleA(db.Model):
+    __tablename__ = 'sched_a'
+
+    sched_a_sk = db.Column(db.Integer, primary_key=True)
+    form_type = db.Column('form_tp', db.String)
+    committee_id = db.Column('cmte_id', db.String)
+    committee_name = db.Column('cmte_nm', db.String)
+    entity_type = db.Column('entity_tp', db.String)
+    contributor_id = db.Column('contbr_id', db.String)
+    contributor_name = db.Column('contbr_nm', db.String)
+    contributor_prefix = db.Column('contbr_prefix', db.String)
+    contributor_first_name = db.Column('contbr_f_nm', db.String)
+    contributor_middle_name = db.Column('contbr_m_nm', db.String)
+    contributor_last_name = db.Column('contbr_l_nm', db.String)
+    contributor_suffix = db.Column('contbr_suffix', db.String)
+    contributor_street_1 = db.Column('contbr_st1', db.String)
+    contributor_street_2 = db.Column('contbr_st2', db.String)
+    contributor_city = db.Column('contbr_city', db.String)
+    contributor_state = db.Column('contbr_st', db.String)
+    contributor_zip = db.Column('contbr_zip', db.String)
+    election_type = db.Column('election_tp', db.String)
+    election_type_full = db.Column('election_tp_desc', db.String)
+    contributor_employer = db.Column('contbr_employer', db.String)
+    contributor_occupation = db.Column('contbr_occupation', db.String)
+    contributor_aggregate_ytd = db.Column('contb_aggregate_ytd', db.Float)
+    contributor_receipt_date = db.Column('contb_receipt_dt', db.DateTime)
+    contributor_receipt_amount = db.Column('contb_receipt_amt', db.Float)
+    receipt_type = db.Column('receipt_tp', db.String)
+    receipt_type_full = db.Column('receipt_desc', db.String)
+    candidate_id = db.Column('cand_id', db.String)
+    candidate_name = db.Column('cand_nm', db.String)
```

Comment:
It sounds similar to dimcandproperties, we decided not to include it at all in the API output because we did not want to give people data that was so inconsistent that it would not be useful. I don't have strong feelings on this. But if it makes the data slightly smaller or faster or cheaper, I would consider taking it out. If it doesn't make a difference, let's not waste your time taking it out.